eternalcore-core/src/main/java/com/eternalcode/core/feature/ignore/IgnoreRepositoryOrmLite.java

```diff
@@ -57,6 +70,25 @@ public CompletableFuture<Boolean> isIgnored(UUID by, UUID target) {
 
     @Override
     public CompletableFuture<Void> ignore(UUID by, UUID target) {
+        Player senderPlayer = this.server.getPlayer(by);
+        Player targetPlayer = null;
+
+        if (!target.equals(IGNORE_ALL)) {
+            targetPlayer = this.server.getPlayer(target);
+        } else {
+            IgnoreAllEvent event = this.caller.callEvent(new IgnoreAllEvent(senderPlayer));
```

Comment:
???????????????????????

Reply:
Omówię to na spotkaniu, chyba nie za bardzo rozumiem

Reply:
Chodzi o to że eventy w repo callujesz